src/index.html

```diff
@@ -13,9 +13,9 @@
     <link rel="stylesheet" href="https://use.typekit.net/zde3qoa.css">
     <link href="https://fonts.googleapis.com/css?family=Poppins|Roboto+Slab:300,400&display=swap" rel="stylesheet">
 
-
+    <!-- 
```

Comment:
Did you mean to leave this commented out?

Reply:
that was left over from an earlier attempt at google auth.  i removed the comment